### PR TITLE
schema breadcrump data in /new en-US only

### PR DESCRIPTION
**Pull request by rraue**
## Description
schema breadcrump data in /new en-US only

## Issue / Bugzilla link
https://github.com/mozilla/bedrock/issues/7699

### CODE REVIEW

File: bedrock/firefox/templates/firefox/new/scene1.html [hunk 1]
```diff
@@ -171,7 +171,9 @@ <h4 class="fxa-title">{{ _('You’ve already got the Firefox browser. Now get ev
 
 {% block newsletter_modal %}{% endblock %}
 
-{% include 'firefox/includes/schemaorg-app.html' %}
+{% if LANG == 'en-US' %}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/new/scene1.html]: It looks like this is in the wrong template. You might want to try `/firefox/new/trailhead/download.html`?

File: bedrock/base/templates/includes/structured-data/organization/mozilla-corporation-organisation.json [hunk 2]
```diff
@@ -0,0 +1,460 @@
+{
```
Comment (alexgibson) [inline bedrock/base/templates/includes/structured-data/organization/mozilla-corporation-organisation.json]: Is this file meant to be included in this PR?

File: bedrock/base/templates/includes/structured-data/organization/mozilla-corporation-organisation.json [hunk 3]
```diff
@@ -0,0 +1,460 @@
...
+    "@context": "http://schema.org",
+    "@type": "Organization",
+    "@id": "https://www.mozilla.org/#organization",
+    "name": "Mozilla",
+    "alternateName": "MoCo",
+    "legalName": "Mozilla Corporation",
+    "description": "Mozilla makes browsers, apps, code and tools that put people before profit. Our mission: Keep the internet open and accessible to all.",
+    "url": "https://www.mozilla.org",
+    "logo": "https://www.mozilla.org/media/img/mozorg/mozilla-256.4720741d4108.jpg",
```
Comment (alexgibson) [inline bedrock/base/templates/includes/structured-data/organization/mozilla-corporation-organisation.json]: We shouldn't be using hashed filenames like this in our JSON. If this image file gets updated, the URL will 404.

Instead I'd suggest using the static image helper, like we do in templates.

```suggestion
    "logo": "{% filter absolute_url %}{{ static('img/mozorg/mozilla-256.jpg') }}{% endfilter %}",
```

File: bedrock/base/templates/includes/structured-data/organization/mozilla-corporation-organisation.json [hunk 4]
```diff
@@ -0,0 +1,460 @@
...
+        "priceCurrency": "USD",
+        "price": "0",
+        "availability": "https://schema.org/InStock"
+        }
+    	},
+    	{
+    "@type": "Product",
+    "@id": "https://www.mozilla.org/#firefoxbeta",
+    "url": "https://www.mozilla.org/en-US/firefox/channel/desktop/",
+    "logo": "https://www.example.com/example-logo.jpg",
```
Comment (alexgibson) [inline bedrock/base/templates/includes/structured-data/organization/mozilla-corporation-organisation.json]: I'm assuming this is just placeholder, but we should not be trusting third party domains for our images here imho.

File: bedrock/firefox/templates/firefox/new/scene1.html (hunk 1 above)
Comment (alexgibson) [reply bedrock/firefox/templates/firefox/new/scene1.html]: I believe if you rebase this branch against master, there should also now be a `{% block structured_data %}` block in the [base templates](https://github.com/mozilla/bedrock/blob/master/bedrock/base/templates/base-pebbles.html#L128-L134) that you can use for positioning here in the template.

File: bedrock/firefox/templates/firefox/includes/schemaorg-breadcrump-new-en-us.html [hunk 5]
```diff
@@ -0,0 +1,27 @@
+<script type="application/ld+json">
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/includes/schemaorg-breadcrump-new-en-us.html]: If you use the new `{% block structured_data %}` block then you can remove the `<script>` tags here and convert this file to `.json`

**Comment by rraue** [comment]
Closing this as I am not able to get it coded myself, needs to get into a sprint and done by a dev.

**Pull request closed without merging**